Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml

```
@@ -539,6 +557,10 @@
           Quantity: 1.5
         - ReagentId: Fiber
           Quantity: 3.5
+  - type: Tag # imp - humans shouldnt be eating moth food
+    tags:
+    - Ingredient
+    - ClothMade
```

Current test failure is my fault :c There's another tag block up above on line 524. Will have to delete that block and add the `Vegan` and `Wheat` tags to this block:
```suggestion
    - ClothMade
    - Vegan # L5
    - Wheat # L5
```